src/image _tags_retriever.py

```diff
@@ -0,0 +1,79 @@
+import json
+import random
+from typing import List
+
+import requests
+
+
+def callapi(key: str = "22499447-34dc1e9873c70161efc9bd199") -> List:
+    """Generates a random image and some descriptors for the same
+
+    :param key: a pixabay.com api access key
+    :return: a list containing two elements - the url of a random image and a list of words describing the image
+    """
+    category = random.choice(
+        [
+            "backgrounds",
+            "fashion",
+            "nature",
+            "science",
+            "education",
+            "feelings",
+            "health",
+            "people",
+            "religion",
+            "places",
+            "animals",
+            "industry",
+            "computer",
+            "food",
+            "sports",
+            "transportation",
+            "travel",
+            "buildings",
+            "business",
+            "music",
+        ]
+    )
+
+    colors = random.sample(
```

Comment:
This should probably be `random.choice` instead.

Reply:
actually I found that using multiple colours in the request yielded a better chance of receiving a result, since many categories don't have vector images in all the colour options